.github/workflows/merge-gate.yml

```
@@ -3,6 +3,12 @@ name: Merge Gate
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)